docs/src/main/asciidoc/amazon-kms.adoc

```diff
@@ -41,7 +41,7 @@ The easiest way to start working with KMS is to run a local instance as a contai
 
 [source,shell,subs="verbatim,attributes"]
 ----
-docker run --rm --name local-kms --publish 8011:4599 -e SERVICES=kms -e START_WEB=0 -d localstack/localstack:0.11.1
+docker run --rm --name local-kms --publish 8011:4599 -e SERVICES=kms -e START_WEB=0 -d localstack/localstack:0.11.5
```

Comment:
I wonder if we can define the version in a common place for reuse?

Reply:
it would make sense.